tests/e2e/postgresql/02-connect.yaml

```diff
@@ -3,7 +3,7 @@ kind: Job
 metadata:
   name: connect-postgresql
 spec:
-  backoffLimit: 1
+  backoffLimit: 6
```

Comment:
Did you adjust this by mistake? The pod itself should not restart anymore, as the check script will simply loop.

Reply:
yes, that's leftover from past solution